runtime-decompiler/src/main/java/org/jrd/frontend/MainFrame/BytecodeDecompilerView.java

```diff
@@ -17,12 +17,7 @@
 import javax.swing.event.DocumentEvent;
 import javax.swing.event.DocumentListener;
 import java.awt.*;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import java.awt.event.ComponentAdapter;
-import java.awt.event.ComponentEvent;
-import java.awt.event.MouseAdapter;
-import java.awt.event.MouseEvent;
+import java.awt.event.*;
```

Comment:
Please don't. Adjust your idea so it do not compact imports and keep enumerated listing.
In idea it is hidden, there is some maximal occurence of imports before the comapcting is used, and that needs to be set to  some 9999